osd_whats_new/osd-whats-new.adoc

```diff
@@ -18,6 +18,8 @@ With its foundation in Kubernetes, {product-title} is a complete {OCP} cluster p
 [id="osd-q2-2024_{context}"]
 === Q2 2024
 
+* **Cluster delete protection.** {product-title} on GCP  users can now enable the cluster delete protection option, which helps to prevent you from accidentally deleting a cluster. For more information, see xref:../osd_install_access_delete_cluster/creating-a-gcp-cluster.adoc#osd-create-gcp-cluster-ccs_osd-creating-a-cluster-on-gcp[Creating a cluster on GCP with CCS].
```

Comment:
```suggestion
* **Cluster delete protection.** {product-title} on {GCP} users can now enable the cluster delete protection option, which helps to prevent users from accidentally deleting a cluster. For more information, see xref:../osd_install_access_delete_cluster/creating-a-gcp-cluster.adoc#osd-create-gcp-cluster-ccs_osd-creating-a-cluster-on-gcp[Creating a cluster on GCP with CCS].
```